packages/webview_flutter/lib/platform_interface.dart

```diff
@@ -10,11 +10,32 @@ import 'package:flutter/widgets.dart';
 
 import 'webview_flutter.dart';
 
+/// Interface for callbacks made by [WebViewPlatform].
+abstract class WebViewPlatformCallbacksHandler {
```

Comment:
Try to understand this class. Does it exist for other packages to override? Does other platform have to override this? What if the other platform doesn't support all these callbacks or needs more callbacks?

Reply:
Extended the comment, let me know if it makes more sense now. I'm also open to rename the class if you have a better suggestion.

A platform that doesn't support a callback will just never invoke it.
New callbacks will have to be added in this package.